### PR TITLE
add getStats promises

### PR DESCRIPTION
**Pull request by fippo**
basically #77 but trying to do one thing at a time.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -288,3 +288,14 @@ test('originalChromeGetStats', function(t) {
...
+test('getStats promise', function(t) {
+  t.plan(2);
+  var pc1 = new m.RTCPeerConnection(null);
+
+  var p = pc1.getStats();
+  t.ok(typeof p === 'object', 'getStats with no arguments returns a Promise');
+
+  var q = pc1.getStats(null);
+  t.ok(typeof q === 'object', 'getStats with a selector returns a Promise');
+});
```
Comment (KaptenJansson) [inline test/test.js:301]: Add another test that test the old style getStats() (i.e. 2 arguments)?

Comment (KaptenJansson) [inline test/test.js:301]: Or Chrome style rather...

Comment (fippo) [inline test/test.js:301]: that's covered by the existing test I think

Comment (KaptenJansson) [inline test/test.js:301]: Yes you are right, https://github.com/fippo/adapter/blob/getstats-promise2/test/test.js#L238.

LGTM.

**Pull request merged**